it/controllers/expenses/UniformsOrToolsExpensesAmountControllerISpec.scala

```diff
@@ -0,0 +1,550 @@
+          formPostLinkCheck(continueLink, formSelector)
+          welshToggleCheck(user.isWelsh)
+        }
+
+        "render 'How much do you want to claim for uniforms, work clothes, or tools?' page with with no pre-filled cya data if amount has not changed" which {
+
+          lazy val result: WSResponse = {
+            dropExpensesDB()
+            authoriseAgentOrIndividual(user.isAgent)
+            userDataStub(userData(fullEmploymentsModel(hmrcExpenses = Some(employmentExpenses(fullExpenses)))), nino, taxYearEOY)
+            insertExpensesCyaData(expensesUserData(isPrior = false, hasPriorExpenses = false,
+              fullExpensesCYAModel), userRequest)
+            authoriseAgentOrIndividual(user.isAgent)
+            urlGet(employmentExpensesAmountPageUrl(taxYearEOY), user.isWelsh, follow = false,
+              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          "has an OK status" in {
+            result.status shouldBe OK
+          }
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(user.commonExpectedResults.expectedCaption)
+          textOnPageCheck(user.specificExpectedResults.get.expectedPreAmountParagraph(300), wantToClaimSelector)
+          buttonCheck(user.commonExpectedResults.continueButtonText, continueButtonSelector)
+          textOnPageCheck(user.commonExpectedResults.expectedCannotClaim, cannotClaimParagraphSelector)
+          textOnPageCheck(hintText, hintTextSelector)
+          textOnPageCheck(poundPrefixText, poundPrefixSelector)
+          inputFieldValueCheck("", amountField)
+          formPostLinkCheck(continueLink, formSelector)
+          welshToggleCheck(user.isWelsh)
+        }
+      }
+    }
+
+    "redirect to another page when the request is valid but they aren't allowed to view the page and" should {
+      val user = UserScenario(isWelsh = false, isAgent = false, CommonExpectedEN, Some(ExpectedIndividualEN))
+
+      "Redirect user to the tax overview page when in year" which {
+
+        lazy val result: WSResponse = {
+          dropExpensesDB()
+          userDataStub(userData(fullEmploymentsModel(hmrcExpenses = Some(employmentExpenses(fullExpenses)))), nino, taxYear)
+          insertExpensesCyaData(expensesUserData(isPrior = false, hasPriorExpenses = false,
+            fullExpensesCYAModel), userRequest)
+          authoriseAgentOrIndividual(user.isAgent)
+          urlGet(employmentExpensesAmountPageUrl(taxYear), user.isWelsh, follow = false, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYear)))
+        }
+
+        "has an SEE_OTHER(303) status" in {
+          result.status shouldBe SEE_OTHER
+          result.header("location") shouldBe Some(s"http://localhost:11111/income-through-software/return/$taxYear/view")
```

Comment:
this can be changed to `Some(appConfig.incomeTaxSubmissionOverviewUrl(taxYear))`